README.md

```diff
@@ -235,7 +235,7 @@ For a full list of all author and/or contributors, check [the contributors page]
 
 The MIT License (MIT)
 
-Copyright (c) 2016-2022 Dealerdirect B.V.
+Copyright (c) 2016-2022 Dealerdirect B.V., PHPCSStandards and contributors
```

Comment:
I think I recall that this should be defined separately:

```
Copyright (c) 2016-2022 Dealerdirect B.V. and contributors
Copyright (c) 2022 PHPCSStandards and contributors

(With subsequent years added on later).
```

**OR** we could request DealerDirect to officially sign over the rights to the new org, then we could just change `Dealerdirect` to `PHPCSStandards` and be done with it.

As this would need @mjrider we could postpone that to later though and just merge this for now.

Reply:
I've updated it per your suggestion for now.

Other than that, let's open an issue about the copyright vs DealerDirect to address that later.